app/src/org/commcare/activities/UpdateActivity.java

```diff
@@ -416,6 +421,17 @@ private static CustomProgressDialog generateNormalUpdateInstallDialog(int taskId
         return dialog;
     }
 
+    private static void reportAppUpdate() {
+        try {
+            String username = CommCareApplication.instance().getRecordForCurrentUser().getUsername();
+            String updateLogMessage = "User " + username + " updated to app version " +
+                    ReportingUtils.getAppBuildNumber();
+            Logger.log(AndroidLogger.TYPE_USER, updateLogMessage);
```

Comment:
Traditionally we use "TYPE_RESOURCES" for logs related to app installs/updates/etc.